repos/c2sm/packages/py-gt4py/package.py

```diff
@@ -19,6 +19,7 @@ class PyGt4py(PythonPackage):
     version('1.0.3.3', tag='icon4py_20240229', git=url)
     version('1.0.3.7', tag='icon4py_20240521', git=url)
     version('1.0.3.8', tag='icon4py_20240711', git=url)
```

Comment:
This version is untested. I recommend to either include it in testing, or remove from the recipe.
No need to carry this on.